### ISSUE TITLE
Event / Prediction not working

### ISSUE THREAD

**Issue by LukeUss** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#309]
After changing to the new API since it would say the streamers did not exist, since then no prediction will go through it tells me that it is either locked or the event is not active anymore. My run.py is the exact same as it was before I changed to the new API.

It is happening to all of the predictions and I know they are active because I would go to the stream and my would not be placed in the prediction which is active

No idea how to reproduce 

OS: Windows 11 (was happening on Windows 10 as well)

This is what it is telling me.
"Failed to place bet, error: EVENT_NOT_ACTIVE"
"Oh no! The event is not active anymore! Current status: LOCKED"

**Comment by lakeroxas3** [comment]
Same here

**Comment by RakSrinaNa** [comment]
This is what Twitch replies. So if it says that it's locked... It probably is.

Locked = prediction is still here and running but bets cannot be placed anymore.

Try modifying your settings and make it better earlier.

**Comment by LukeUss** [comment]
I added delay=300 into the code within the streamersettings section and it still wont place any bets i get the same 2 outcomes 
as stated in the OP. I have tried changing the delay to 200 and 100 and the bet is still not being placed.

It was able to place one bet within the past few days only and I don't know why that was the only one that went through. 

It only started to do this when I changed to the different API

**Comment by RakSrinaNa** [comment]
The example from the readme probably won't solve the issue, you'll have to setup a custom delay for the predictions for the streamer that has issues (eg: `place prediction 1 mins after the beginning` rather than `x seconds before the end`).

**Comment by Berksmb** [comment]
The streamer does short bets, 60 seconds.
I can confirm it's working 3/3 bets were placed.

**Comment by LukeUss** [comment]
So the way I fixed it was reinstalling the whole thing and using the example from within the main page. The example I was using is from a while ago and it didn't have the bet delay added to it. I added it to my original run.txt and it didn't even start or when I got it to start it still didn't place any bets. 

So if it isn't working for you guys try and reinstall everything and use the example given.
